app/buffer_manager.py

```diff
@@ -94,6 +91,7 @@ def loadTextBuffer(self, relPath):
         app.log.info('creating a new file at\n ', fullPath)
       textBuffer = app.text_buffer.TextBuffer()
       self.renameBuffer(textBuffer, fullPath)
+      textBuffer.view = view
```

Comment:
I still have hope for allowing multiple views to access a textBuffer at the same time. So setting a particular view on a textBuffer will conflict with that. It wasn't enough of an issue to give up the rest of this PR. I just wanted to mention this bit about someday having a many to one relationship between views and a textBuffer.